src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -70,11 +70,27 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
             => ColumnSelectingEstimator.DropColumns(CatalogUtils.GetEnvironment(catalog), columnsToDrop);
 
         /// <summary>
-        /// ColumnSelectingEstimator is used to select a list of columns that user wants to drop from a given input.
+        /// ColumnSelectingEstimator is used to select a list of columns that user wants to keep from a given input.
```

Comment:
>keep [](start = 92, length = 4)

shouldn't this be keep or drop?

Reply:
Just keep. SelectColumns only specifies those columns to keep.